.github/workflows/snapshots.yml

```diff
@@ -0,0 +1,31 @@
+name: Update Snapshots
+
+on:
+  workflow_dispatch:
+
+jobs:
+  update_snapshots:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Fail if branch is main
+        if: ${{ github.ref == 'refs/heads/main' }}
+        run: |
+          echo "This workflow should not be triggered with workflow_dispatch on main"
+          exit 1
+
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: npm ci
+      - name: Install Playwright Browsers
+        run: npx playwright install --with-deps chromium
+      - name: Run Playwright tests
+        run: npm run e2e:snapshots
+      - name: Commit Playwright updated snapshots
+        uses: EndBug/add-and-commit@v9
+        # We don't want to commit formatting changes to main
```

Comment:
What does formatting have to do with it?

Reply:
Copy/paste, good catch. Thx